features/step_definitions/assessment_steps.rb

```diff
@@ -4,12 +4,14 @@
 
 When('I choose to schedule an assessment') do
   @assessment_type = 'schedule'
-  click_link 'Schedule an assessment'
+  page.find('#assessmentDropdownElements').click
```

Comment:
Shouldn't this be clicking `#btnAssessmentDropdown` to expand the dropdown? And wouldn't that only work on a `@javascript` enabled test anyway?